.gitignore

```diff
@@ -7,3 +7,8 @@
 NESW_Textadventure/Properties/AssemblyInfo.cs
 NESW_Textadventure/obj/*
 *.pdb
+NESW_Textadventure/NESW_Textadventure - Copy/Properties/AssemblyInfo.cs
```

Comment:
Überlege dir mal, warum du diese zusätzlichen Einträge in deiner .github Datei hast